Cybersecurity_Tools/ClipCrypt/clipcrypt/core.py

```
@@ -0,0 +1,291 @@
+"""
+Core ClipCrypt module.
+
+Main interface for the ClipCrypt clipboard manager.
+"""
+
+import os
+import sys
+from pathlib import Path
+from typing import List, Dict, Any, Optional, Tuple
```

Ruff (0.12.2)

`typing.List` imported but unused

Remove unused import

(F401)